index.js

```diff
@@ -29,6 +29,7 @@ module.exports = (params) => {
     bqEventsTableName: 'events', // name of the BigQuery table that dfe-analytics streams event data into
     bqEventsTableNameSpace: null, // optional - value of the namespace field in the events table to filter by. Use this to distinguish between multiple applications or interfaces which stream events to the same events table.
     eventsDataFreshnessDays: 1, // Number of days after which, if no new events have been received, the events_data_not_fresh assertion will fail to alert you to this
+    eventsDataFreshnessDisableDuringRange: false, // whether to disable the events_data_not_fresh assertion if today's date is currently between one of the ranges in assertionDisableDuringDateRanges
```

Comment:
To check my own understanding: When we choose to activate this assertion, will this be set to a formula that checks if todays date is within the ranges described in 'assertionDisableDuringDateRanges'